modules/FIC_FileHandler.py

```diff
@@ -1,10 +1,13 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from modules.FIC_DataVault import FICDataVault
+import json
 
 
-class FICFileHandler:
+class FICFileHandler(FICDataVault):
     def __init__(self):
+        FICDataVault.__init__(self)
         from modules.config import CHANGELOG_JSON_PATH, CHANGELOG_MD_PATH, CHANGELOG_REPO_PATH
 
         self.changelog_json = CHANGELOG_JSON_PATH
```

Comment:
Don't think we need this, since we can access the `CHANLOG_JSON_PATH` anywhere inside the class.
Could I be missing some edge case on why we need it?